dev/export-consul.ctmpl

```diff
@@ -2,6 +2,9 @@
 set -e
 
 function log_deprecated(){
+    if [ "${LOG_CONSUL_DEPRECATION}" != "true" ]; then
+        return 1
```

Comment:
with `set -e`, this is considered an error and will exit

Reply:
Why did my brain tell me to use a 1?! :doh: fixing to return 0 which seems to be what I want. Thanks.

Reply:
You can also just use `return`